src/FlightBookingApp.Application/Features/Auth/Commands/IdentityService.cs

```diff
@@ -121,7 +103,7 @@ private async Task<TokenModel> GenerateJWTTokenWithUserClaimsAsync(User user)
                 issuer: _authSettingsOptions.Issuer,
                 audience: _authSettingsOptions.Audience,
                 claims: claims,
-                expires: DateTime.Now.AddDays(1),
+                expires: DateTime.Now.AddDays(2),
```

Comment:
You have expiration time setting in authConfig, you shouldn't hardcode this value